src/utils/constants.ts

```diff
@@ -0,0 +1,11 @@
+export const NETWORKS = {
+  ARBITRUM_RINKEBY: '421611',
+  ARBITRUM: '42161',
+  MAINNET: '1',
+  RINKEBY: '4',
+  KOVAN: '42'
+} as const
```

Comment:
was this as const required??

Reply:
not required but it forces the ts compiler to type the object more strictly.

ie without `as const` this will be typed as 

```
{
  ARBITRUM_RINKEBY: string,
  ARBITRUM: string,
  MAINNET: string,
  RINKEBY: string,
  KOVAN: string
}
```

but with it, its typed as

```
{
  ARBITRUM_RINKEBY: '421611',
  ARBITRUM: '42161',
  MAINNET: '1',
  RINKEBY: '4',
  KOVAN: '42'
}
```

Reply:
ahhh nice, learn something new everyday